specs/_features/maxeb_increase/capella.py

```diff
@@ -2334,9 +2347,14 @@ def get_slot_signature(state: BeaconState, slot: Slot, privkey: int) -> BLSSigna
     return bls.Sign(privkey, signing_root)
 
 
-def is_aggregator(state: BeaconState, slot: Slot, index: CommitteeIndex, slot_signature: BLSSignature) -> bool:
+def is_aggregator(state: BeaconState, slot: Slot, index: CommitteeIndex, validator_index: ValidatorIndex, slot_signature: BLSSignature) -> bool:
+    validator = state.validators[validator_index]
     committee = get_beacon_committee(state, slot, index)
-    modulo = max(1, len(committee) // TARGET_AGGREGATORS_PER_COMMITTEE)
+    min_balance_increments = validator.effective_balance // MIN_ACTIVATION_BALANCE
+    committee_balance = get_total_balance(state, set(committee))
+    denominator = committee_balance ** min_balance_increments
+    numerator = denominator - (committee_balance -  TARGET_AGGREGATORS_PER_COMMITTEE * MIN_ACTIVATION_BALANCE) ** min_balance_increments
```

Comment:
closing bracket seems to be missing

Reply:
not sure which you are referring to. i think all are accounted for (and my linter does too)

Reply:
Oh, my bad! I did notice to spaces here: `  TARGET_AGGREGATORS_PER_COMMITTEE`